### PR TITLE
(XW-1969) Build compiled CSS file to distribute

### PR DESCRIPTION
**Pull request by Grunny**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -92,6 +92,17 @@ gulp.task('svg-individual', function () {
 gulp.task('svg', ['svg-sprite', 'svg-individual']);
 
 
+// BUILD DESIGN SYSTEM STYLES
+gulp.task('build-styles', function () {
+	return gulp.src('dist/styles.scss')
+		.pipe(scss())
+		.pipe(autoprefixer({
+			browsers: ['last 3 versions'],
```
Comment (kvas-damian) [inline gulpfile.js]: I see mercury is using different config for supported browsers: https://github.com/Wikia/mercury/blob/4eb062496b96876656dde7c29147582638a10727/front/auth/gulpfile.js#L45
Do we need to do something about it?

File: gulpfile.js [hunk 2]
```diff
@@ -92,6 +92,17 @@ gulp.task('svg-individual', function () {
...
+			cascade: false
+		}))
+		.pipe(gulp.dest('dist/css'));
```
Comment (kvas-damian) [inline gulpfile.js]: Is it possible to minimize output file?

Comment (Grunny) [inline gulpfile.js]: Done in 4a7ef490cbd4221ff8dc95158075586dbd23f574.

File: gulpfile.js (hunk 1 above)
Comment (Grunny) [inline gulpfile.js]: I'm not sure. This just matches what we're doing for reference page. Someone with better knowledge on why the config options were chosen for both RP and Mercury would be able to make a call.

**Comment by kvas-damian** [comment]
What happened to VRT that we had to update baseline in this PR?
I don't what we changed on a Reference Page.

**Comment by Grunny** [comment]
I'm not sure, I didn't do that. Was it cherry-picked? Nothing should have changed on the reference page. I can rebase against master and get rid of that commit to clean it up.

**Comment by kvas-damian** [comment]
LGTM

**Comment by rogatty** [comment]
I've updated the VRT baseline. Something changed with VM / Selenium / Chrome or whatever and currently VRT fails for mobile version on all branches.

**Comment by rogatty** [comment]
LGTM

**Pull request merged**